core/src/modules/ops/record.ts

```diff
@@ -502,8 +551,11 @@ export namespace RecordOps {
         const { canImport, properties } = await source.import(record);
 
         // We need to save each property as it is loaded so it can be used as a mapping for the next source
+        // We also don't want to save more than one recordProperty at a time so we can isolate any problem values
         if (canImport && toSave) {
-          await addOrUpdateProperties([record], [properties], false);
+          for (const [k, property] of Object.entries(properties)) {
+            await addOrUpdateProperties([record], [{ [k]: property }], false);
+          }
```

Comment:
This split here is slower, but allows us to retry just one failing write of a RecordProperty.  This method isn't used in the schedule/run workflow so it is OK to slow it down here